src/core/reducers/api.js

```diff
@@ -2,23 +2,16 @@
 import UAParser from 'ua-parser-js';
 
 import { LOG_OUT_USER } from 'amo/reducers/users';
-import {
-  SET_AUTH_TOKEN,
-  SET_LANG,
-  SET_CLIENT_APP,
-  SET_REQUEST_ID,
-  SET_USER_AGENT,
-} from 'core/constants';
 import type { LogOutUserAction } from 'amo/reducers/users';
-import type {
-  SetAuthTokenAction,
-  SetClientAppAction,
-  SetLangAction,
-  SetRequestIdAction,
-  SetUserAgentAction,
-} from 'core/actions/index';
 import type { Exact } from 'core/types/util';
 
+export const SET_AUTH_TOKEN = 'SET_AUTH_TOKEN';
+export const SET_CLIENT_APP = 'SET_CLIENT_APP';
+export const SET_LANG = 'SET_LANG';
+export const SET_REGION_CODE = 'SET_REGION_CODE';
+export const SET_REQUEST_ID = 'SET_REQUEST_ID';
+export const SET_USER_AGENT = 'SET_USER_AGENT';
```

Comment:
I thought we had to specify a type for each action like:

```js
export const SET_USER_AGENT: 'SET_USER_AGENT' = 'SET_USER_AGENT';
```

Otherwise `typeof SET_USER_AGENT` will be `string`.

Reply:
This is unchanged code, simply copied from one place to another, so it works, and Flow doesn't complain, but you're right, we generally do this, so I'll fix it.